examples/lambda-api-gateway/package.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "lambda-api-gateway",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
```

Add scripts to align with other examples

```suggestion
  "license": "MIT",
  "scripts": {
    "build": "...",
    "dev": "...",
    "test-dev": "...",
    "test-start": "...",
  },
```